proposals/NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md

```diff
@@ -0,0 +1,103 @@
+# Addition of the way to notify HMI of the required support for apps
+
+* Proposal: [SDL-NNNN](NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Java Suite / HMI / RPC / Protocol ]
+
+## Introduction
+This proposal is to provide a way to notify HMI of the required support for apps by using the new notification messages. 
+
+
+## Motivation
+If the system does not have something to support required for apps, apps may not work properly or be able to register. For example, in the case of Android OS, if a Media app connects via only USB, since the system cannot establish the audio path, Media app cannot play audio and does not register arbitrarily. At the result, it cannot allow user to know the reason why app cannot be registered. To solve these problems, we propose to add a new notification message to let HMI know the support the SDL app needs. By doing so, this will allow HMI to display the appropriate message and apps connectivity and operability will be improved.
+
+
+## Proposed solution
+To provide a way to notify HMI of the required support for apps, we will add new notification messages `Request Additional Support` and `Ping Apps` on Protocol spec and new RPCs `RequestAdditionalSupport` and `PingApps` on HMI_API.
```

Comment:
```suggestion
To provide a way to notify the HMI of the required support for apps, we will add new notification messages `Request Additional Support` and `Ping Apps` to the Protocol Spec and new RPCs `RequestAdditionalSupport` and `PingApps` to the HMI_API.
```

Reply:
Same comment as above. Please use "...required support for apps," in this context.

Reply:
@theresalech -san,
We reworded "required support for apps" in this context and made a commit them.
Thank you.